src/components/tooltip/tooltip.style.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'preact-emotion';
+
+export const TooltipWrapper = styled('div')`
+  position: relative;
+`;
+
+export const TooltipSpan = styled('span')`
+  background: rgba(0, 0, 0, 0.5);
+  border-radius: 5px;
+  color: #fff;
```

Comment:
Could we import the `colors` file here and use `colors.white` instead? 🙏

Reply:
the `style` of the `tooltip`, does not have access to the `theme`, so this would have to be passed by each level. Still, it is possible yes.

Reply:
I tried adding white color to the background of the tooltip, I did not think it was cool.